docs/source/project_documentation/aac_gen_gherkin.md

```diff
@@ -0,0 +1,84 @@
+# AaC Gen-Gherkin
+
+The AaC Gen-Gherkin plugin generates gherkin feature files based on provided AaC models.
+
+## gen-gherkin-behaviors Command
+
+```bash
+aac gen-gherkin-behaviors architecture-file.aac output/directory
+```
+
+### Arguments
+
+#### Architecture File
+The AaC file containing the `model` definition/s.
+
+#### Output Directory
+The directory in which the gherkin feature files will be generated.
+
+## Definition Type References
+The `gen-gherkin` plugin will generate feature files based on the `behavior` field of `model` definitions.
+`behavior` definitions also have a field named `acceptance`.  The  `acceptance` field is where the feature files scenario steps will come from.
+
+### Example Model Definition
+
+```yaml
+model:
+    name: example_name
+    description: Description of Model
+    behavior:
+        - name: example_behavior
+          description: Does something
+          acceptance:
+            - name: example_acceptance
+              scenarios:
+                - name: example scenario
+                  given:
+                    - Initial state
+                  when:
+                    - Something happens
+                  then:
+                    - The model does something in response.
+```
+
+### Gherkin Syntax
+When output to a feature file, a `model`s `behavior` becomes a `feature`, and its `acceptance` becomes a `scenario`.
+Each `scenario` has a number of steps:
+`given`: An optional step, but is usually recommended to include.  The `given` step describes the models initial state.
+`when`: This step describes what kicks off the action the `model` will perform, be it called through a command line or triggered some other way.
+`then`: This step describes what happens after the `model` performs its action.  Typically it will include whatever output is created by the `model`'s action.
+
+Along with the `scenario` steps, `features` and `scenario`s can also have requirements, signified by the `requirement ID` before the `feature` or `scenario`, such as the following:
+```
```

Comment:
Since this is the block quote it will not handle correctly for me to apply an actual suggestion.

But should have something like ```text to remove the highlighting since there isn't special syntax highlighting for the output in the feature files.